base58/README.md

```diff
@@ -0,0 +1,27 @@
+Bitcoin base58 encoding
+=======================
+
+This crate provides encoding and decoding of base58 strings as defined by the Bitcoin network
```

Comment:
To me "Bitcoin network" means mostly P2P which doesn't really use base58. I'd prefer "Bitcoin ecosystem".